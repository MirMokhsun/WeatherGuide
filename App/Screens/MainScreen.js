import React, { Component } from 'react'
import {  ImageBackground, View, ScrollView } from 'react-native'
import { styles } from '../styles/mainScreenStyles'
import Header from '../components/header'

const Wallpaper = require('../assets/Images/Wallpaper.jpg')
export default class MainScreen extends Component {
    render() {
        return (
            <ImageBackground style={styles.container}
                resizeMode='cover'
                source={Wallpaper} >
                < Header/>
                <ScrollView style={styles.container}>
                    <View style={styles.WB}></View>
                    <View style={styles.WBH}></View>
                    <View style={styles.WBD}></View>
                    <View style={styles.WBD}></View>
                </ScrollView>
            </ImageBackground>
        )
    }
}