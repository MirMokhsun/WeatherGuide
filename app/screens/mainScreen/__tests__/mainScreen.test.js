import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import MainScreen from '..';

describe('Main', () => {
    test('Main snapshot', () => {
        const renderer = new ShallowRenderer();
        const snap = renderer.render(<MainScreen />);
        expect(snap).toMatchSnapshot();
    });
});