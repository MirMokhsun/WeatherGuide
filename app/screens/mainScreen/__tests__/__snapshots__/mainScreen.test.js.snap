// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main Main snapshot 1`] = `
<ImageBackground
  resizeMode="cover"
  source={
    Object {
      "testUri": "../../../app/assets/images/Wallpaper.jpg",
    }
  }
  style={
    Object {
      "flex": 1,
    }
  }
>
  <ScrollViewMock
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <View
      style={
        Object {
          "backgroundColor": "rgba(0,0,0,0.3)",
          "borderRadius": 3,
          "height": 125,
          "marginLeft": 5,
          "marginTop": 15,
          "width": 740,
        }
      }
    />
    <View
      style={
        Object {
          "backgroundColor": "rgba(0,0,0,0.3)",
          "borderRadius": 3,
          "height": 150,
          "marginLeft": 5,
          "marginTop": 25,
          "width": 740,
        }
      }
    />
    <View
      style={
        Object {
          "backgroundColor": "rgba(0,0,0,0.3)",
          "borderRadius": 3,
          "height": 150,
          "marginLeft": 5,
          "marginTop": 15,
          "width": 740,
        }
      }
    />
    <View
      style={
        Object {
          "backgroundColor": "rgba(0,0,0,0.3)",
          "borderRadius": 3,
          "height": 150,
          "marginLeft": 5,
          "marginTop": 15,
          "width": 740,
        }
      }
    />
  </ScrollViewMock>
</ImageBackground>
`;
